tests/unit/controller/HttpErrorTest.php

```diff
@@ -72,12 +74,32 @@ public function providesExceptionData() {
 	 * @param String $status
 	 */
 	public function testJsonError($exception, $message, $status) {
-		$httpError = $this->getMockForTrait('\OCA\Gallery\Controller\HttpError');
+		$request = $this->createMock(IRequest::class);
```

Comment:
OK, but it's failing on all versions, so there must be something else. I'll try locally ASAP.